nixos/modules/services/misc/dendrite.nix

```diff
@@ -1,12 +1,12 @@
 { config, lib, pkgs, ... }:
 let
-  cfg = config.services.matrix-dendrite;
+  cfg = config.services.dendrite;
   settingsFormat = pkgs.formats.yaml { };
   configurationYaml = settingsFormat.generate "dendrite.yaml" cfg.settings;
-  workingDir = "/var/lib/matrix-dendrite";
+  workingDir = "/var/lib/dendrite";
```

Comment:
Is this a breaking change? Is there manual intervention necessary to move the certs and such?

Reply:
Theoretically not, because the paths to `tlsKey` etc. have to be specified manually. However, this will practically break configuration because the module is broken in a way that forces the keys to be within the `workingDir`.
Oh and user data will have to be moved manually as well.

Anyway, I think this is fine because the module has only been around for a few days.